docs/source/etcd-data-model.rst

```diff
@@ -40,8 +40,19 @@ endpoints
   (e.g. if it has multiple vNICs). See :ref:`endpoint-data` for more.
 
 security profiles
-  A security profile is an object that encapsulates a specific security policy
-  that can be applied to endpoints. See :ref:`security-profile-data` for more.
+  A security profile is an object that encapsulates a specific set of security
+  rules to apply to an endpoint.  Endpoints reference one or more security
+  profiles explicitly.  See :ref:`security-profile-data` for more.
+
+security policies
+  Similarly, security policies contain a set of security rules to apply.
+  However, security policies provide a richer, tiered security model, which
```

Comment:
I'd prefer s/provide/allow/
I'd delete 'richer', as the richness is fully expressed by 'tiered' and 'override', so it's redundant to say 'richer' as well.